src/engine/Purse.java

```diff
@@ -0,0 +1,43 @@
+package engine;
+
+import view.Board;
+
+public class Purse 
+{
+	private int points = 0;
+	private Board board;
+	
+	
+	public Purse(Board board)
+	{
+		this.board = board;
+	}
+	
+	/**
+	 * accumulate points for given turn	 
+	 */
+	public void accumulatePoints()
+	{
+		points += 10; //default point accumulation
```

Comment:
The amount of points per turn is unique per level in LevelInfo. Thus, this class needs a levelInfo reference, or it needs to handoff the "finding out how much to add" responsibility to game engine